index.bs

```diff
@@ -210,76 +210,77 @@ XRHand {#xrhand-interface}
 ------
 
 <pre class="idl">
+enum XRHandJoint {
+  "wrist",
+
+  "thumb-metacarpal",
+  "thumb-phalanx-proximal",
+  "thumb-phalanx-distal",
+  "thumb-tip",
+
+  "index-finger-metacarpal",
+  "index-finger-phalanx-proximal",
+  "index-finger-phalanx-intermediate",
+  "index-finger-phalanx-distal",
+  "index-finger-tip",
+
+  "middle-finger-metacarpal",
+  "middle-finger-phalanx-proximal",
+  "middle-finger-phalanx-intermediate",
+  "middle-finger-phalanx-distal",
+  "middle-finger-tip",
+
+  "ring-finger-metacarpal",
+  "ring-finger-phalanx-proximal",
+  "ring-finger-phalanx-intermediate",
+  "ring-finger-phalanx-distal",
+  "ring-finger-tip",
+
+  "pinky-finger-metacarpal",
+  "pinky-finger-phalanx-proximal",
+  "pinky-finger-phalanx-intermediate",
+  "pinky-finger-phalanx-distal",
+  "pinky-finger-tip"
+};
+
 [Exposed=Window]
 interface XRHand {
-    iterable&lt;XRJointSpace>;
-    readonly attribute unsigned long length;
-    getter XRJointSpace joint(unsigned long jointIndex);
-
-    const unsigned long WRIST = 0;
-
-    const unsigned long THUMB_METACARPAL = 1;
-    const unsigned long THUMB_PHALANX_PROXIMAL = 2;
-    const unsigned long THUMB_PHALANX_DISTAL = 3;
-    const unsigned long THUMB_PHALANX_TIP = 4;
-
-    const unsigned long INDEX_METACARPAL = 5;
-    const unsigned long INDEX_PHALANX_PROXIMAL = 6;
-    const unsigned long INDEX_PHALANX_INTERMEDIATE = 7;
-    const unsigned long INDEX_PHALANX_DISTAL = 8;
-    const unsigned long INDEX_PHALANX_TIP = 9;
-
-    const unsigned long MIDDLE_METACARPAL = 10;
-    const unsigned long MIDDLE_PHALANX_PROXIMAL = 11;
-    const unsigned long MIDDLE_PHALANX_INTERMEDIATE = 12;
-    const unsigned long MIDDLE_PHALANX_DISTAL = 13;
-    const unsigned long MIDDLE_PHALANX_TIP = 14;
-
-    const unsigned long RING_METACARPAL = 15;
-    const unsigned long RING_PHALANX_PROXIMAL = 16;
-    const unsigned long RING_PHALANX_INTERMEDIATE = 17;
-    const unsigned long RING_PHALANX_DISTAL = 18;
-    const unsigned long RING_PHALANX_TIP = 19;
-
-    const unsigned long LITTLE_METACARPAL = 20;
-    const unsigned long LITTLE_PHALANX_PROXIMAL = 21;
-    const unsigned long LITTLE_PHALANX_INTERMEDIATE = 22;
-    const unsigned long LITTLE_PHALANX_DISTAL = 23;
-    const unsigned long LITTLE_PHALANX_TIP = 24;
+    iterable&lt;XRHandJoint, XRJointSpace>;
```

Comment:
When defining something as iterable, [per Web IDL](https://heycam.github.io/webidl/#idl-iterable):

> Prose accompanying an interface with a pair iterator must define a list of value pairs for each instance of the interface, which is the list of value pairs to iterate over.

Reply:
@domenic Thanks! I updated the PR. Can you take another look?

Reply:
LGTM